d2.config.js

```diff
@@ -20,6 +20,8 @@ const config = {
     },
 
     coreApp: true,
+
+    skipPluginLogic: true,
```

Comment:
Why is this necessary?

Reply:
Good catch, info added to the PR description.